lib/simpler/controller.rb

```diff
@@ -43,12 +53,16 @@ def render_body
     end
 
     def params
-      @request.params
+      @request.env['simpler.params']
```

Comment:
Если установить это в App, тут было бы
```suggestion
      @request.params.merge(@request.env['simpler.params'])
```